src/Domain/ValueObjects/NullICO.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DDDWorkshop\Domain\ValueObjects;
+
+class NullICO extends ICO
```

Comment:
We didn't speak about `NullICO` with domain expert. What is the motivation for this new concept? What problems it solves? Is it necessary?

This is the only really problematic part of your code in my opinion (but it doesn't immediately mean it is wrong)

Reply:
Well, user can Issue invoices for someone who does not have ICO and is identified only by Name, At least that is what i remember from the lectures. But it is possible that my brain had made that up. If we are gonna issue invoices only to buyers with ICO, than i will remove it.

Reply:
Also, reason for namespaces is just my preference. I like having stuff separated and in my mind the VO, Entities, Services etc. are different enough to warrant a separate place to put them.

Reply:
* `NullICO` - I thought that `null` could be suitable for these use cases... But in the other hand, returning `ICO` is much better that returning `?ICO`.... hm... I'll have to think about it more
* namespaces - ok, separation for easier thinking is fine